Gruntfile.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(grunt){
       },
       dist: {
         files: {
-          'index.js': ['bower_components/opal/opal/current/opal.js']
+          'index.js': ['bower_components/opal/opal/0.7.1/opal.js']
```

Comment:
Tag not yet available so the `current` directory is still referencing the `v0.6.3`